src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -74,7 +74,7 @@ file that was distributed with this source code.
                 ajax: {
                     url:  '{{ url ?: path(route.name, route.parameters|default([])) }}',
                     dataType: 'json',
-                    delay: {{ delay === 100 and quiet_millis !== 100 ? quiet_millis : delay }}, // NEXT_MAJOR: Replace by `{{ delay }}` instead.
+                    delay: {{ delay == 100 and quiet_millis !== 100 ? quiet_millis : delay }}, // NEXT_MAJOR: Replace by `{{ delay }}` instead.
```

Comment:
I guess `!==` also does not work? let me try

Reply:
indeed

Reply:
fixed